index.js

```diff
@@ -143,15 +145,15 @@ const handleToggle = () => {
 
 deleteTaskInputCard.addEventListener("click", resetTaskInput);
 
-toggleButton.addEventListener("click", () => handleToggle(todos));
+toggleButton.addEventListener("click", handleToggle);
 
-initialTaskContainer.addEventListener("click", () => handleToggle(todos));
+initialTaskContainer.addEventListener("click", handleToggle);
 
-addTaskButton.addEventListener("click", handleCreateTodo);
+addTaskButton.addEventListener("click", () => handleCreateTodo(todos));
```

Comment:
Let's move all the event listener function to `element.js` where we are targeting the dom. Their respective event listener should be in `action.js`. 

The idea is that, our index.js will be clean, minimal and very easy to understand.